config/default.json

```diff
@@ -2,5 +2,7 @@
   "mongoURI": "mongodb+srv://ammar123:ammar123@hackyoursocial-dwlwn.mongodb.net/test?retryWrites=true",
   "jwtSecret": "mysecrettoken",
   "githubClientId": "de50a87630d9ed45b81d",
-  "githubSecret": "b1a2ddad1fa5e0aa0495737ac99cf0766ac56a8e"
+  "githubSecret": "b1a2ddad1fa5e0aa0495737ac99cf0766ac56a8e",
+  "sendgrid": "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",
+  "verificationSecret": "verification-ZSI6IjVkMGQ3YWUzMDJiZjczNGI3NDMzZWQ2NDE1NjE2NjQxMTkyMzEifSwia"
 }
```

Comment:
We can hide these keys and secret into `process.env` and we can retrieve them in anywhere.
Here is an explanation how to do it:

https://medium.com/codait/environment-variables-or-keeping-your-secrets-secret-in-a-node-js-app-99019dfff716